src/main/java/net/minecraftforge/event/TickEvent.java

```diff
@@ -66,6 +67,25 @@ public WorldTickEvent(LogicalSide side, Phase phase, Level world)
             this.world = world;
         }
     }
+
+    /**
+     * Called before and after a chunk is ticked for things like random ticks and weather. Useful for random-tick and weather based actions.
+     */
+    public static class EnvironmentTickEvent extends TickEvent
+    {
+        public final Level level;
```

Comment:
Is there a reason why this is a `Level` rather than a `ServerLevel`, since this event is fired from `ServerChunkCache`?